cns/restserver/api.go

```diff
@@ -1522,3 +1522,43 @@ func (service *HTTPRestService) nmAgentSupportedApisHandler(w http.ResponseWrite
 
 	logger.Response(service.Name, nmAgentSupportedApisResponse, resp.ReturnCode, serviceErr)
 }
+
+// getVMUniqueID retrieves VMUniqueID from the IMDS
+func (service *HTTPRestService) getVMUniqueID(w http.ResponseWriter, r *http.Request) {
+	logger.Printf("[Azure CNS] getVMUniqueID")
```

Comment:
Does this log line add any value? We are logging the request on the next line

Reply:
removed it. seems redundant.